crest/Assets/Crest/Crest/Scripts/Collision/QueryBase.cs

```diff
@@ -38,7 +38,7 @@ public abstract class QueryBase : MonoBehaviour
         const int s_computeGroupSize = 64;
         public static bool s_useComputeCollQueries = true;
 
-        readonly static int sp_queryPositions_minGridSizes = Shader.PropertyToID("_QueryPositions_MinGridSizes");
+        int sp_queryPositions_minGridSizes = Shader.PropertyToID("_QueryPositions_MinGridSizes");
```

Comment:
Don't we still want to keep these as readonly?

Reply:
I assumed that we need to reinit them each time. I did google it but didnt find anything. so i made them not readonly so they can be re-init'd each time. I think this is the safest route.

Reply:
sorry just realised this was one i removed the static flag from. yeah they could be readonly, good point. ill take a look

Reply:
done on latest. good catch